express/assets/javascripts/show-password-two-fields.js

```diff
@@ -0,0 +1,85 @@
+window.GOVUK = window.GOVUK || {};
```

Comment:
This file is identital to [show-password.js](https://github.com/alphagov/di-onboarding-self-service-experience/blob/b64ef29d6a7a0aa49579231f03381f46f43448ef/express/assets/javascripts/show-password.js). We should make the existing file/function reusable instead of duplicating big chunks of code.

Reply:
The file is not identical, it is similar. And it handles the second hide/show button. Some of the code might be re-usable - we can create  a tech debt for it. This was the quickest working solution to enable TWO show/hide password buttons on a same page.